_includes/cta-omero-iviewer.html

```diff
@@ -0,0 +1,17 @@
+<!-- begin CTA -->
+<hr class="whitespace">
+<header class="marketing-hero marketing-hero-omero-figure">
+    <div class="row">
+        <div class="small-12 columns bg-image-text-container-cta text-center">
+            <h2>Power of the web</h2>
+            <p>
+                Using OpenLayers technology, get all the functionality of our
+                desktop client in the next generation web-based image viewer
```

Comment:
Import is a popular functionality of our desktop client.

Reply:
I'll add 'visualization and annotation'